objectherkenning_openbare_ruimte/data_sampling/decos_helper.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import pathlib
+
+import geopandas as gpd
+import pandas as pd
+
+RD_CRS = "EPSG:28992"
```

Comment:
Could you add an inline comment to describe what's RD_CRS? Even just the full name of the CRS should be enough